README.md

```diff
@@ -70,7 +71,49 @@ Before beginning a sprint:
 
 During a sprint we track progress using a [scrum or kanban-like board](#notes)<sup>4</sup> with status columns for To Do, In Progress, Blocked, Ready for Review, Ready for QA, and Done. 
 
-After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs. 
+After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs.
+
+## Velocity Tracking
+
+### What is Velocity Tracking?
+
+Velocity tracking is the practice of using our average historical output to inform how we plan and commit to upcoming work. 
+
+Velocity is a number that represents the amount of work completed by a team during a specific time period. Typically, we talk about “sprint velocity” and “average sprint velocity”. `sprint-velocity` describes the amount of work completed in a single sprint. `average-sprint-velocity` is an average of the previous *n* sprints (where *n* is something like 2 - 4 sprints). We use the latter during sprint planning to determine how much work to commit to in the sprint being planned.
+
+### How Do We Track Velocity?
+
+Velocity tracking stuff mostly occurs during sprint planning. 
+
+If the sprint you're planning is the first sprint where you'll be incorporating anything related to velocity you need to choose an initial "fake" `average-sprint-velocity`. Its value might be something like 0.5 (avg. number of points completed per person per hour).
```

Comment:
Instead of using fake, we should use a different term ("seed" as Nicole mentioned might be better). Additionally I think this should be a note that we don't reference again. This seed is just a temporary average since there is no average to use.
It could be "Note: if this is your first time velocity tracking, use a seed value of .5 to get your estimations started. By the end of your first sprint, you'll have a real velocity to base your future sprints off of, at that point you can stop using the fake seed number."

This also might tie into Nicole's suggestion of making it clear how the cycle works.